src/Scene.js

```diff
@@ -15,6 +15,61 @@ type SceneProps = {
 };
 
 class Scene extends React.Component<SceneProps, {}> {
+
+    drawGrid(numRow: number, numColumn: number, width: number, minX: number, minY: number) {
+        const grid = [];
+        let xOffset = minX;
+        let yOffset = minY;
+        for (let i=1;i < numRow + 1;i++) {
+            yOffset = yOffset + width;
+            if (i < numRow) {
+            grid.push(<line
+                className='Scene__grid-line'
+                key={`grid-cell-row-${i}`}
+                x1={`${xOffset}`}
+                y1={`${yOffset}`}
+                x2={`${-xOffset}`}
```

Comment:
This calculation for the grid line endpoint assumes that the origin is in the middle. It is at the moment, but it will break if we change where the origin is (and because the calculation of the Scene bounds are in another function (`render`), we could break it without realizing there is a connection). The math could be generalized by passing in the Scene width and height (or calculating from the number of rows and columns and grid cell width), and using that.

Together with the suggestion above of using `minX` Instead of `xOffset`, we would use:

    x1={`${minX}`}
    y1={`${yOffset}`}
    x2={`${minX + sceneWidth}`}
    y2={`${yOffset}`}